spras/allpairs.py

```diff
@@ -13,7 +14,7 @@
 
 
 class AllPairs(PRM):
-    required_inputs = ['nodetypes', 'network']
+    required_inputs = ['nodetypes', 'network', 'directed_flag']
```

Comment:
I don't see an updated config.yaml file or config.py file that sets the directed_flag.

Reply:
This isn't required. Take, for example, [DOMINO's `active_genes`](https://github.com/Reed-CompBio/spras/blob/215de99d1533a404d50633961d7ada1a5f94369b/spras/domino.py#L30), which is unspecified in `config.yaml`.